test/focus-trap-react.test.js

```diff
@@ -317,6 +323,156 @@ describe('FocusTrap', () => {
         expect(document.body).toHaveFocus();
       });
     });
+
+    it('Does not call onPostActivate() until checkCanFocusTrap() has completed', async () => {
+      let hasActivated = false;
+      let hasRunPostActivate = false;
```

Comment:
Can we use mock functions instead these flags?
```
const onActivate = jest.fn();
const onPostActivate = jest.fn();
```
set these as the options, and then later,
```
expect(onActivate).toHaveBeenCalled();
```

Reply:
Oh, I didn't know about mock functions. Yeah that will probably work as long as there is also `expect(onActivate).toNotHaveBeenCalled()`

Reply:
Yes, but it should be `expect(onActivate).not.toHaveBeenCalled()`